webapp/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -2159,6 +2159,38 @@
     </module>
   </portlet>
 
+  <portlet>
+    <name>SpaceCreation</name>
+    <module>
+      <script>
+        <minify>false</minify>
+        <path>/js/spaceCreation.bundle.js</path>
+      </script>
+      <depends>
+        <module>commonVueComponents</module>
+      </depends>
+      <depends>
+        <module>vue</module>
+      </depends>
+      <depends>
+        <module>vuetify</module>
+      </depends>
+      <depends>
+        <module>eXoVueI18n</module>
+      </depends>
+      <depends>
+        <module>jquery</module>
+        <as>$</as>
+      </depends>
+      <depends>
+        <module>extensionRegistry</module>
+      </depends>
+      <depends>
+        <module>spaceForm</module>
+      </depends>
```

Comment:
If you call to this module using an explicit RequireJS call, than it's no more necessary to mark it as a dependency to the portlet Module

```javascript
      window.require(['SHARED/spaceForm'], drawer => drawer.open(false, true));
```

```suggestion
```